src/pkg/projects/windowsdesktop/src/windowsdesktop.depproj

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+
+  <PropertyGroup>
+    <FrameworkPackageName>Microsoft.WindowsDesktop.App</FrameworkPackageName>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.WindowsDesktop.App">
```

Comment:
Shouldn't you instead be picking up the packages that make this thing up and replacing it?  That way we can delete dotnet-trusted.

Reply:
I was going for an incremental approach for now, but I'll try moving that over here in case it's not much worse to get it all done at the same time.

Reply:
I see.  That's fair, but make sure you understand what's going on over there so you can see the path forward and what needs to be shared vs not.  You'll find that a lot of stuff happening in dotnet-trusted overlaps with stuff that's going on here.

Reply:
It turns out there's DotNet-Trusted changes I need first to make all the bits available--filed https://github.com/dotnet/core-setup/issues/5219 to track this. I'm going to look a bit more at this in case there's something I'm missing but I don't plan to block this PR on that.

Reply:
Sounds good.  Thanks for looking into it and identifying the work.